README.md

```
@@ -1,3 +1,71 @@
 # java-janggi
 
 장기 미션 저장소
+
+## 기능 구현
+
+### 장기(장기판), Janggi
+
+- [x] 생성 시 장기판을 초기화한다.
+- [x] 장기판의 위치를 선택하면, 해당 장기말이 움직일 수 있는 모든 경로를 반환한다.
+- [x] 장기말을 장기판의 위치를 이용해 선택하고, 도착지를 지정해 해당 장기말을 움직일 수 있다.
+    - [x] 도착지가 규칙에 벗어나면 예외를 발생시킨다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)